### PR TITLE
update use of Buffer

### PR DESCRIPTION
**Pull request by kevinhuang12345678**

### CODE REVIEW

**Comment by kevinhuang12345678** [comment]
issue: https://github.com/msiebuhr/node-statsd-client/issues/90

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### v0.4.6 (2021-04-19)
+
+- update usage of Buffer
+
```
Comment (papandreou) [inline CHANGELOG.md:4]: This file is autogenerated, so it's not necessary to update it as part of the PR. No worries, though, as it'll be overwritten when we make the next release :)

File: lib/EphemeralSocket.js [hunk 2]
```diff
@@ -175,11 +175,12 @@ EphemeralSocket.prototype._send = function _send(data) {
...
         // Create message
-        var message = new Buffer(data);
+        var message = Buffer.from(data);
+        var msg = message.toString();
 
-        debug(message.toString());
+        debug(msg);
 
-        that._socket.send(message, 0, message.length, that._port, that._hostname);
+        that._socket.send(msg, 0, msg.length, that._port, that._hostname);
```
Comment (papandreou) [inline lib/EphemeralSocket.js:183]: Why include this unrelated change? It introduces a bug for non-ascii messages where the string length isn't equal to the (serialized) buffer length.

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "udp",
     "tcp"
   ],
-  "version": "0.4.5",
+  "version": "0.4.6",
```
Comment (papandreou) [inline package.json:12]: Please leave out the version bump, that'll be done as part of the release.

File: package-lock.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "statsd-client",
-  "version": "0.4.5",
+  "version": "0.4.6",
```
Comment (papandreou) [inline package-lock.json:3]: Please leave out the version bump, that'll be done as part of the release.

**Review by papandreou — CHANGES_REQUESTED** [review]

File: lib/EphemeralSocket.js (hunk 2 above)
Comment (kevinhuang12345678) [reply lib/EphemeralSocket.js:183]: can you update the branch? thanks.

Comment (papandreou) [reply lib/EphemeralSocket.js:183]: If you push a fix to the branch in your fork, this PR will automatically get updated 🤗

File: CHANGELOG.md (hunk 1 above)
Comment (kevinhuang12345678) [reply CHANGELOG.md]: cleanned.

File: lib/EphemeralSocket.js (hunk 2 above)
Comment (kevinhuang12345678) [reply lib/EphemeralSocket.js]: updated.

File: package.json (hunk 3 above)
Comment (kevinhuang12345678) [reply package.json]: cleanned.

File: package-lock.json (hunk 4 above)
Comment (kevinhuang12345678) [reply package-lock.json]: cleanned.

**Review by papandreou — APPROVED** [review]
LGTM, and congratulations 🎉

**Comment by papandreou** [comment]
Released in 0.4.6
